config/action.d/pf.conf

```diff
@@ -75,14 +75,13 @@ protocol = tcp
 # Option:  port
 # Notes.:  specifies port to monitor
 # Values:  [ NUM | STRING ]  Default:
-#
-#port = telnet
+port = ssh
```

Comment:
I think, the port (as well as protocol) defined in action self, should't be used at all, because there are defined in definition section (that normally expected `%(port)s` substitution) and currently both parameters will be substituted using tag syntax `<port>`, that are external options (or options from init section).